.github/workflows/azure-static-web-app.yml

```diff
@@ -40,13 +42,13 @@ jobs:
         id: builddeploy
         uses: Azure/static-web-apps-deploy@v1
         with:
-          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN }}
+          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_NICE_GRASS_068870F1E }}
```

Comment:
Same as the comment on the API deployment but this one uses the deployment token.  Again, it might be unnecessary to change.

Reply:
Yeah, still probably better safe than sorry I guess. Presuming it doesn't break the temperamental little balance of the app's whole universe.